### ISSUE TITLE
Login Button on FF does not work

### ISSUE THREAD

**Issue by drkthng** [AugurProject/augur#801]
**Repro:**
Firefox 49.0.2
Go to Login Page (ex: https://augur-dev.firebaseapp.com/?page=login)
Type in your credentials
Click Login Button

**Expectation:**
User gets logged in.

**Actual:**
Nothing happens, no login.

**Possible reason:**
Button has attribute disabled="" which might be interpreted the wrong way by FF.
No Problems on Chrome.
After deleting this attribute via the DevTools Login works also on FF.

**Comment by tinybike** [comment]
Hm, I just tried this in Firefox (same version) and it worked fine.  Are you sure you entered the correct login ID?  (The "login ID" is the very very long string that you copy/pasted during registration -- you can see it if you go to the Account page in the UI.)

**Comment by drkthng** [comment]
As said in the Incident-Report, I also tried it on Chrome -> worked! and I tried in on FF after removing the disabled attribute altogether -> worked!
Which FF version did you use?

**Comment by tinybike** [comment]
FF 49.0.2, the same version you're using!  Would you mind re-trying it in FF?  (And, if it doesn't work again, could you take a screenshot of the login screen?)

**Comment by drkthng** [comment]
I'm on it!

**Comment by drkthng** [comment]
Hey tinybike! you're right the phenomenon is not there anymore! I also get logged in automatically after "saving" the credentials via the browser, which wasn't the case yesterday, I had to login every time I restarted the browser (thought this was due to ongoing development). 
Seems a lot has changed, since now I also see the markets I need to report on.
Long story short, Incident doesn't exist atm. Can be closed.
Cheers!
